sdk/purview/purview-catalog-rest/test/public/utils/recordedClient.ts

```diff
@@ -10,7 +10,7 @@ import { ClientOptions } from "@azure-rest/core-client";
 import { env, Recorder, RecorderStartOptions } from "@azure-tools/test-recorder";
 
 const replaceableVariables: { [k: string]: string } = {
-  ENDPOINT: "https://endpoint",
+  ENDPOINT: "https://endpoint/",
```

Comment:
Should we add this change in the code generator ?

Reply:
Will verify this!